### PR TITLE
Add Amethyst logo to sponsors

### PR DESCRIPTION
**Pull request by Wodann**

### CODE REVIEW

File: src/pages/index.scss [hunk 1]
```diff
@@ -166,6 +166,11 @@
     background-color: $color1;
 
     #sponsors > div {
-        padding: 40px;
+        box-sizing: border-box;
```
Comment (baszalmstra) [inline src/pages/index.scss]: netlify requires at least 40px paddings around the logo

**Comment by baszalmstra** [comment]
https://www.netlify.com/press/#badges

File: src/pages/index.scss (hunk 1 above)
Comment (Wodann) [reply src/pages/index.scss]: Fixed.

Comment (Wodann) [reply src/pages/index.scss]: Fixed

**Pull request merged**